spring-data-sample/README.md

```diff
@@ -0,0 +1,260 @@
+# Spring Data integration Sample
+
+This is a sample application for Spring Data integration with ScalarDB.
+
+## Prerequisites
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+## Sample application
+
+### Overview
+
+This article describes how to create an applicaiton for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) using Spring Data integration with ScalarDB.
```

Comment:
The main difference from `scalardb-sample`'s README are this `Overview` and command output examples (e.g. this application uses Jackson which doesn't put a space after comma)